PRESUBMIT.py

```
@@ -0,0 +1,145 @@
+        command,
+        stdout=input_api.subprocess.PIPE,
+        stderr=input_api.subprocess.PIPE,
+        encoding='utf-8')
+    return handle_result(out[0], out[1], retcode)
+  except input_api.subprocess.CalledProcessError as e:
+    errors = []
+    if explain_error:
+      errors += explain_error(e)
+    errors += [
+        output_api.PresubmitPromptOrNotify(
+            'Command "%s" returned exit code %d. Output: \n\n%s' %
```

Ruff (0.14.13)

Use format specifiers instead of percent format

(UP031)